src/middlewared/middlewared/plugins/smb.py

```diff
@@ -368,18 +368,6 @@ async def configure(self, job, create_paths=True):
         job.set_progress(20, 'Wait for ix-netif completion.')
         await self.netif_wait()
 
-        """
-        Since some NSS modules will default to setting home directory to /var/empty,
-        verify that this path is immutable during setup for SMB service (prior to
-        initializing directory services).
-        """
```

Comment:
Looks like this is now being done in account.py `setup`.  Maybe move this comment there (and adapt)?